packages/docs/src/styles/components/navigation.scss

```diff
@@ -0,0 +1,111 @@
+// Mobile
+.c-navigation {
+  background-color: $color-white;
+  width: 100%;
+
+  .c-navigation__header {
+    background-color: $color-primary-alt-lightest;
```

Comment:
`primary-alt` is now `secondary` so anywhere you use primary alt you can substitute with `secondary` going forward.